package.json

```diff
@@ -21,10 +31,10 @@
     "babel-cli": "^6.24.1",
     "babel-core": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1"
+    "babel-preset-stage-1": "^6.24.1",
+    "postinstall-build": "^5.0.1"
```

Comment:
I tried `yarn install` on a fresh clone and it worked fine (yarn 0.24.5, npm 3.10.3, node v6.7.0). `postinstall-build` anyhow has to be in the `dependencies` to function.

I could also just remove it, it is only there so people can directly depend on a source tree (i.e. using `git+https://....../redux-query-sync.git#commit` as package dependency), letting the dependency be built upon installation; a feature I would probably like to see baked into npm itself.

Anyhow, this is a different issue, let's not have it in this PR.